### ISSUE TITLE
Why the following packages “are not available from current channels”? How to solve them?

### ISSUE THREAD

**Issue by luminous-wu** [gbattocletti-riccardoUrb/RL_exploration_for_UAVs#1]
- args==0.1.0
  - winshell==0.6
  - yaspin==2.1.0
  - google-auth==2.3.3
  - pillow==9.0.0
  - sentry-sdk==1.5.1
  - tensorboard-plugin-wit==1.8.0
  - shortuuid==1.0.8
  - fonttools==4.28.5
  - uff==0.6.9
  - configparser==5.2.0
  - google-auth-oauthlib==0.4.6
  - tensorboard==2.7.0
  - gitpython==3.1.24
  - pipwin==0.5.1
  - gym==0.21.0
  - graphsurgeon==0.4.5
  - typing_extensions==4.0.1
  - sklearn==0.0
  - tensorflow-gpu==2.7.0
  - tensorflow-estimator==2.7.0
  - setuptools==60.2.0
  - pywin32==303
  - numpy==1.22.0
  - pypiwin32==223
  - pytz-deprecation-shim==0.1.0.post0
  - grpcio==1.43.0
  - kneed==0.7.0
  - keras==2.7.0
  - pyttsx3==2.90
  - jaraco.context==4.1.1
  - pysmartdl==1.3.4
  - tifffile==2022.2.9
  - comtypes==1.1.11
  - absl-py==1.0.0
  - clint==0.5.1
  - wandb==0.12.9
  - opencv-python==4.5.5.62
  - gitdb==4.0.9
  - pyparsing==3.0.6
  - pyjokes==0.6.0
  - imutils==0.5.4
  - wikipedia==1.4.0
  - cachetools==4.2.4
  - tzdata==2021.5
  - wolframalpha==5.0.0
  - rsa==4.8
  - opt-einsum==3.3.0
  - pyjwt==2.3.0
  - onnx-graphsurgeon==0.2.6
  - imageio==2.16.1
  - tensorflow-io-gcs-filesystem==0.23.1
  - threadpoolctl==3.0.0
  - twilio==7.7.1
  - tzlocal==4.1
  - smmap==5.0.0
  - pyprind==2.11.3
  - charset-normalizer==2.0.9
  - importlib-metadata==4.10.0
  - markdown==3.3.6
